pkg/admin/run_check.go

```diff
@@ -0,0 +1,25 @@
+package admin
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"k8s.io/klog/v2"
+)
+
+func microshiftShouldNotRun() error {
+	cmd := exec.Command("sh", "-c", "ps aux | grep  -v grep | grep 'microshift run'")
```

Comment:
Is this better done by running `pidof` command? Note that we should also check microshift-etcd.
```
pidof microshift microshift-etcd && echo Running
```

Reply:
Unfortunately `pidof` also includes pid of `microshift admin backup`

Reply:
Should we instead use systemctl? `systemctl show --no-pager microshift` produces machine readable format.

Reply:
For pidof, we could also use `--omit-pid` option for the current PID of the caller. In any case, we should call something more robust than `ps` command, possibly `systemctl` that Pablo suggested.

Reply:
I wanted to check for actual process, not systemd unit - ensure on a more general "linux process level" than just systemd

Reply:
But if we don't want  such strong guarantee, then we can go with what Pablo suggested